R/module_teal_data.R

```diff
@@ -89,14 +97,16 @@ srv_teal_data <- function(id,
 
     srv_validate_reactive_teal_data(
       "validate",
-      data = try_module_out,
+      teal_data_r = try_module_out,
       modules = modules,
       validate_shiny_silent_error = validate_shiny_silent_error,
       hide_validation_error = is_previous_failed
     )
   })
 }
 
+srv_teal_data <- srv_teal_data_module
```

Comment:
so inside code this is still `srv_teal_data`?

Reply:
Yes, I made this decision to make it easier to keep the code as is before evaluating how much code should be changed if we replaced srv_teal_data by srv_teal_data_module